cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -220,21 +259,26 @@
     "methodology": {
       "properties": {
         "description": {
+          "description": "A description of the methodology used for the production of this data/software",
           "type": "string"
         },
         "links": {
           "items": {
             "properties": {
               "description": {
+                "description": "A brief description of the link associated with the methodology",
                 "type": "string"
               },
               "recid": {
+                "description": "If the link is to another Open Data record, the record id of that record",
```

Comment:
"another CERN Open Data portal record"  would be more precise regarding internal vs external open data links